src/Microsoft.Sbom.Parsers.Spdx30SbomParser/Parser/ContextsResult.cs

```diff
@@ -2,17 +2,17 @@
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
 using System.Collections.Generic;
-using System.Linq;
 using Microsoft.Sbom.JsonAsynchronousNodeKit;
 
 namespace Microsoft.Sbom.Parser;
 
 public record ContextsResult : ParserStateResult
 {
-    public ContextsResult(ParserStateResult result)
+    public ContextsResult(ParserStateResult result, List<object> jsonList)
         : base(result.FieldName, result.Result, result.ExplicitField, result.YieldReturn)
     {
+        Contexts = jsonList;
     }
 
-    public IEnumerable<string> Contexts => ((IEnumerable<string>)this.Result!).Select(r => r);
+    public IEnumerable<object> Contexts { get; set; }
```

Comment:
`object` can literally be anything. Is there a way to improve the type safety?

Reply:
I've changed the ContextsResult to add more type safety by making Contexts a List<string> instead of List<object>